apps/prairielearn/elements/pl-sketch/pl-sketch.py

```diff
@@ -0,0 +1,36 @@
+import chevron
+import lxml.html
+import prairielearn as pl
+
+
+def prepare(element_html, data):
+    return data
+
+
```

Comment:
This is a no-op and should be removed.

Reply:
As Eliot pointed out in his review, you should indeed be validating attributes here.